resources/assets/less/app.less

```diff
@@ -384,6 +384,23 @@ a.logo.no-hover a:hover {
   background-color: transparent;
 }
 
+.index-blocks {
```

Comment:
I don't think this should be pluralized and I also wonder if there is a better name than `index-block` 🤔

Reply:
Agreed on the name. We can also stack classes, so IMHO this class shouldn't really have more than 

```
white-space: nowrap;
overflow: hidden;
text-overflow: ellipsis;
```

and should inherit the colors and other styles of the original thing it's added to.

Reply:
do we have suggestion for the name? these are the tiles on the settings index page, so....
setting-tile? tile-block?

the original was help-block, so i was going off that.

Reply:
help-block is built into bootstrap. You can chain the classes using both help-block and whatever we call the ellipses one, so you don’t have to mess with colors. Name could be contrast-card-blocks maybe? Dunno. We only use them there, but that could also change over time.